exercises/grains/grains_test.exs

```diff
@@ -16,57 +16,57 @@ defmodule GrainsTest do
 
   # @tag :pending
   test "square 1" do
-    assert Grains.square(1) === 1
+    assert Grains.square(1) === { :ok, 1 }
   end
 
   @tag :pending
   test "square 2" do
-    assert Grains.square(2) === 2
+    assert Grains.square(2) === { :ok, 2 }
   end
 
   @tag :pending
   test "square 3" do
-    assert Grains.square(3) === 4
+    assert Grains.square(3) === { :ok, 4 }
   end
 
   @tag :pending
   test "square 4" do
-    assert Grains.square(4) === 8
+    assert Grains.square(4) === { :ok, 8 }
   end
 
   @tag :pending
   test "square 16" do
-    assert Grains.square(16) === 32768
+    assert Grains.square(16) === { :ok, 32768 }
   end
 
   @tag :pending
   test "square 32" do
-    assert Grains.square(32) === 2147483648
+    assert Grains.square(32) === { :ok, 2147483648 }
   end
 
   @tag :pending
   test "square 64" do
-    assert Grains.square(64) === 9223372036854775808
+    assert Grains.square(64) === { :ok, 9223372036854775808 }
   end
 
   @tag :pending
   test "total grains" do
-    assert Grains.total === 18446744073709551615
+    assert Grains.total === { :ok, 18446744073709551615 }
   end
 
   @tag :pending
   test "square greater than 64 raises an exception" do
-    catch_error Grains.square(65)
+    assert Grains.square(65) === { :error, "The requested square must be between 1 and 64 (inclusive)" }
```

Comment:
I wondered about whether or not these tests should care what the error message is.  They probably could also be written something like
```
{ response_type, mesage } = Grains.square(65)
assert(response_type) === :error
```

Also, I'm still new to elixir so if there is a more succinct way of pattern matching that `===` with just the `:error`, just give me a pointer on how that would be done.